src/Feature.ts

```diff
@@ -0,0 +1,69 @@
+export type FeatureFlag = {
+    [internalName: string]: boolean;
+};
+
+/**
+ * @todo documentation
+ *
+ * @since {date}
+ */
+export class Feature {
+    static readonly TASK_STATUS_MENU = new Feature(
```

Comment:
Aaaahhh, this is more granular than I first thought... It's very specifically to the status menu.

I had wrongly assumed that the Feature would be 'Enable new task states UI'...
Hence my other comments about enabling the command and the GUI changes and putting them behind the one feature flag.

So that's why a bunch of other code, about settings and commands, is commented out.

Reply:
Yes, I added flags very late in the cycle so it is for work post the status change all up, the UI will be covered by it and the append global filter capability will behind it as well which is in a separate PR.